quay-entrypoint.sh

```diff
@@ -104,6 +100,28 @@ case "$QUAYENTRY" in
         done
         exec supervisord -c "${QUAYCONF}/supervisord.conf" 2>&1
         ;;
+    "repomirror-nomigrate")
+        echo "Entering repository mirroring mode"
+        export QUAY_SERVICES="${QUAY_SERVICES}${QUAY_SERVICES:+,}repomirrorworker,pushgateway"
+        echo "Running services ${QUAY_SERVICES}"
+        for f in "${QUAYCONF}"/init/*.sh; do
+            if [ "$f" != "/quay-registry/conf/init/runmigration.sh" ]; then
+                echo "Running init script '$f'"
+                "$f" || exit
```

Comment:
Should this have `ENSURE_NO_MIGRATION=true` as well, like `registry-nomigrate`?

Reply:
yes, thanks